### PR TITLE
MineOnline.MineOnline version 4.8.8

### PR DESCRIPTION
**Pull request by daisyUniverse**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96220)

### CODE REVIEW

**Comment by robinuniverse** [comment]
@microsoft-github-policy-service agree

**Review by Trenly — CHANGES_REQUESTED** [review]
Since this is a repackaged exe, I don't believe we can accept it, given that it doesn't come from the original publisher

**Comment by robinuniverse** [comment]
> Since this is a repackaged exe, I don't believe we can accept it, given that it doesn't come from the original publisher

Would it make it kosher if I were to set the source of the MSI download from the MineOnline repo itself? I am a contributor and could probably swing it, but I didn't want to gum up the original repo

**Review by OverRevvv — APPROVED** [review]

**Comment by stephengillie** [comment]
This package requires a version of the Java Runtime Environment, possibly from the package `Oracle.JavaRuntimeEnvironment`, to run. 
image

**Comment by robinuniverse** [comment]
> This package requires a version of the Java Runtime Environment, possibly from the package `Oracle.JavaRuntimeEnvironment`, to run. image

Looks like I still have more to learn about packaging MSI's! I will see about including a JRE redist in the binary as part of the install process next weekend and report back on my progress here. thank you for your time 👍

File: manifests/m/MineOnline/MineOnline/4.8.8/MineOnline.MineOnline.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MineOnline.MineOnline
+PackageVersion: 4.8.8
```
Comment (stephengillie) [inline manifests/m/MineOnline/MineOnline/4.8.8/MineOnline.MineOnline.installer.yaml:5]:
```suggestion
PackageVersion: 4.8.8
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```

**Pull request closed without merging**